teamscale_precommit_client/precommit_client.py

```diff
@@ -201,6 +225,10 @@ def _format_message(self, finding):
 
         return '%s | (%s)' % (message, link)
 
+    def _remove_path_prefix_from_finding(self, finding):
+        finding_without_path_prefix = copy.deepcopy(finding)
```

Comment:
Why do we need a copy here? Maybe explain in a comment?

Reply:
If I have a method taking a parameter and returning a parameter of the same type I'd always expect it not to modify the input. 
I find a comment explaining such a basic principle a bit verbose here. 

Note: Technically, we probably even could modify the input finding in this case, but I wanted to avoid weird side effects in the future.

Reply:
I see you point. I'd suggest to rename the method to something like `_without_path_prefix(finding)` then? I'm honestly not sure what a conventional name in Python would be, just thinking about how to express the behaviour more clearly in the names...